sdks/python/apache_beam/runners/worker/log_handler.py

```diff
@@ -162,7 +162,7 @@ def _read_log_control_messages(self):
     # This case is unlikely and the chance of reconnection and successful
     # transmission of logs is also very less as the process is terminating.
     # I choose not to handle this case to avoid un-necessary code complexity.
-    while self._alive:
+    while True:
```

Comment:
My suggestion for easier readability (take it or leave it):
```py
alive = True  # Force at least one connection attempt.
while alive:
  ...
  alive = self._alive
```

Reply:
Yes, that does read better. Thanks.